src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,260 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static Calculator calculator;
+    private static final double VALUE1 = 34.324;
+    private static final double VALUE2 = -23.21;
+    private static final double DELTA = 0.0001;
+    private static final double ZERO = 0;
+    private static final char PLUS = '+';
```

Comment:
```suggestion
    private static final char ADDITION = '+';
```
use consitent naming of operators

Reply:
add